rgw/admin/caps.go

```diff
@@ -0,0 +1,56 @@
+package admin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// AddUserCap will add the capabilities for a user
+func (api *API) AddUserCap(ctx context.Context, uid, userCap string) ([]UserCapSpec, error) {
```

Comment:
Should we have an `UpdateUserCap()` method? Or perhaps it should be `SetUserCap()` instead of `AddUserCap()`?

Reply:
@BlaineExe SetQuota() overwrites existing values to the new one, but here it just adds this list to existing caps.
For example, lets say initial Cap was {"users:read"}, if  AddUserCap() call with cap {"user:write", "bucket:read"}, now new cap set on the user will be {"user:*","bucket:read"}.

For UpdateUserCap() query please see https://github.com/ceph/go-ceph/pull/571#discussion_r705338198